apis/management.cattle.io/v3/rke_types.go

```diff
@@ -804,6 +809,10 @@ type MonitoringConfig struct {
 	Options map[string]string `yaml:"options" json:"options,omitempty"`
 	// NodeSelector key pair
 	NodeSelector map[string]string `yaml:"node_selector" json:"nodeSelector,omitempty"`
+	// Update strategy
+	UpdateStrategy *appsv1.DeploymentStrategy `yaml:"update_strategy" json:"updateStrategy,omitempty"`
+	// Number of monitoring addon pods
+	Replicas *int32 `yaml:"replicas" json:"replicas,omitempty" norman:"default=2"`
```

Comment:
This is a change from the current config, why was configured with 2 replicas?

Reply:
Having more than one replica will be good for zero-downtime use case when the metrics-server deployment undergoes rolling update.